BlueprintUI/Tests/StackTests.swift

```diff
@@ -1176,13 +1176,162 @@ class StackTests: XCTestCase {
             ],
             expectedSizes: [
                 StackLayout.Vector(axis: 10, cross: 10),
-                StackLayout.Vector(axis: 20, cross: 50), // 2 x 5
-                StackLayout.Vector(axis: 60, cross: 20), // 6 x 2
+                StackLayout.Vector(axis: 27, cross: 50), // 2 x 5
+                StackLayout.Vector(axis: 53, cross: 20), // 6 x 2
```

Comment:
Agree. My view on current system:

(a) I do not like the current system, and dare I say no one (outside of like 3 people, and 2 of us are on this PR) understand it (I do not like green eggs and ham)
(b) Every time I see someone trying to use a non-binary priority (eg 0 or 1), it's used wrong
(c) I think this change actually makes things more understandable / behave as you expect for the common case – eg because we're now measuring flexible elements in the actual size they have, it fixes many of the issues called out in https://github.com/square/Blueprint/pull/265 – which you can see from running this in Market – the weird "huh, that row layout sure is wrong" bugs are fixed.

I will update the tests here, but I'd also like to follow up with turning growing and shrinking into bools vs ints, since, yeah...